src/hal/getpinmap_ttgo_lora32_v2.1.cpp

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
Please look at the changes I made for #736 -- it turns out that compilers don't actually do a great job of discarding vtables, so we need to condition this module compilation on `defined(ARDUINO_TTGO_LoRa32_v21new)`.